web/beacon-app/src/constants/rolesAndStatus.ts

```diff
@@ -6,8 +6,10 @@ export const MEMBER_ROLE = {
 };
 
 export const MEMBER_STATUS = {
-  CONFIRMED: 'Confirmed',
+  CONFIRMED: 'Confirmed', // remove this when we have the new API
   PENDING: 'Pending',
+  ONBOARDING: 'Onboarding',
+  ACTIVE: 'Active',
 };
```

Comment:
I got this from @pdeziel last PR :

https://github.com/rotationalio/ensign/blob/0b98cbc7f086216020ed53c1a16bbe0e640c467f/pkg/tenant/db/members.go#L50